cmd/kni-install/main.go

```diff
@@ -34,13 +35,30 @@ func main() {
 	// https://github.com/openshift/installer/pull/1478
 	rand.Seed(time.Now().UTC().UnixNano())
 
-	if len(os.Args) > 0 {
+	if argLen := len(os.Args); argLen > 0 {
+		// Support external terraform plugins
 		base := filepath.Base(os.Args[0])
 		cname := strings.TrimSuffix(base, filepath.Ext(base))
 		if pluginRunner, ok := plugins.KnownPlugins[cname]; ok {
 			pluginRunner()
 			return
 		}
+
+		// Support internal terraform plugins
+		if argLen > 3 {
+			if os.Args[1] == "internal-plugin" {
+				pluginType := os.Args[2]
+				plugin := os.Args[3]
+
+				switch pluginType {
+				case "provisioner":
```

Comment:
This is based on https://github.com/hashicorp/terraform/blob/v0.12.1/command/internal_plugin.go#L48.  `provisioner` is the only thing supported by `internal-plugin` at the moment